src/status_im2/data_store/wallet.cljs

```diff
@@ -0,0 +1,25 @@
+(ns status-im2.data-store.wallet
+  (:require
+    clojure.set))
+
+(defn <-rpc
+  [network]
+  (-> network
+      (clojure.set/rename-keys
+       {:Prod                   :prod
+        :Test                   :test
+        :isTest                 :is-test
```

Comment:
It would be nice if this was renamed to `test?` because it will lead to more idiomatic code when destructuring the key.

Reply:
good point, will adjust this and other booleans :)